src/hooks/useEthPrice.tsx

```diff
@@ -1,15 +1,13 @@
 import { useEffect, useState } from "react"
 
-import { getData } from "../utils/cache"
-
 export const useEthPrice = (): number => {
   const [ethPrice, setEthPrice] = useState<number>(0)
   useEffect(() => {
     ;(async () => {
       try {
-        const data = await getData<{ ethereum: { usd: number } }>(
+        const data: { ethereum: { usd: number } } = await fetch(
           "https://api.coingecko.com/api/v3/simple/price?ids=ethereum&vs_currencies=usd"
-        )
+        ).then((res) => res.json())
```

Comment:
If we're using async/await I'd opt for avoiding the `.then` chaining here... would consider:

```ts
const response = await fetch("...")
if (!response.ok) throw new Error("error message here")
const { ethereum: { usd } } = response.json()
```